File: app/src/main/java/com/getnway/hellokotlin/java/MainActivity.java
package com.getnway.hellokotlin.java;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import com.getnway.hellokotlin.R;

public class MainActivity extends AppCompatActivity {
    private EditText et_content;
    private TextView tv_content;
    private Button btn_ok;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        et_content = (EditText) findViewById(R.id.et_content);
        tv_content = (TextView) findViewById(R.id.tv_content);
        btn_ok = (Button) findViewById(R.id.btn_ok);

        btn_ok.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                tv_content.setText(et_content.getText());
            }
        });
    }
}

File: app/src/main/java/com/getnway/hellokotlin/kotlin/MainActivity.kt
package com.getnway.hellokotlin.kotlin

import android.app.Activity
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.util.Log
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import com.getnway.hellokotlin.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
//    private var et_content: EditText? = null
//    private var tv_content: TextView? = null
//    private var btn_ok: Button? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
//        et_content = findViewById(R.id.et_content) as EditText
//        tv_content = findViewById(R.id.tv_content) as TextView
//        btn_ok = findViewById(R.id.btn_ok) as Button

//        btn_ok!!.setOnClickListener { tv_content!!.text = et_content!!.text }
        btn_ok.setOnClickListener { tv_content.text = et_content.text }
    }
}
